src/tools/populateLegacyResourceTitlesCli.js

```diff
@@ -0,0 +1,28 @@
+import readline from 'readline';
+import moment from 'moment';
+import processLegacyResources from './populateLegacyResourceTitles';
+import { DATE_FORMAT } from '../constants';
+import { auditLogger } from '../logger';
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+rl.question('Please enter a START date in the format MM/DD/YYYY (ENTER for -30 days):', async (startDate) => {
+  const start = !startDate ? moment().subtract(30, 'd') : moment(startDate);
+  if (!start.isValid()) {
+    auditLogger.error(`Invalid START date, please use formate MM/DD/YYYY: ${startDate}`);
+  }
+  rl.question('Please enter a END date in the format MM/DD/YYYY (ENTER for today):', async (endDate) => {
+    const end = !endDate ? moment().subtract() : moment(endDate);
```

Comment:
```suggestion
    const end = !endDate ? moment().subtract() : moment(endDate, 'MM/DD/YYYY');
```

Reply:
adding these format spec will stop the noise in the command window (date is not in a recognized ISO format, etc)